cmake-proxies/cmake-modules/AudacityFunctions.cmake

```diff
@@ -227,7 +227,10 @@ function( audacity_append_common_compiler_options var use_pch )
          PRIVATE
             # include the correct config file; give absolute path to it, so
             # that this works whether in src, modules, libraries
-            $<$<PLATFORM_ID:Windows>:/FI${CMAKE_BINARY_DIR}/src/private/configwin.h>
+            $<$<PLATFORM_ID:Windows>:
+               $<$<CXX_COMPILER_ID:MSVC>:/FI${CMAKE_BINARY_DIR}/src/private/configwin.h>
```

Comment:
I think `$<IF>` makes more sense here. This way we separate MSVC and GCC/CLANG

Reply:
> I think `$<IF>` makes more sense here. This way we separate MSVC and GCC/CLANG

Excuse me... I'm sorry but I have not understood very well how I should use the `$<IF>` expression here.

Reply:
Something like

```
$<IF:$<CXX_COMPILER_ID:MSVC>, 
    /FI${CMAKE_BINARY_DIR}/src/private/configwin.h,
    -include ${CMAKE_BINARY_DIR}/src/private/configwin.h
>
```